awscli/examples/cloudwatch/wait/composite-alarm-exists.rst

```diff
@@ -0,0 +1,7 @@
+**To wait until a composite alarm exists**
+
+The following ``wait composite-alarm-exists`` example pauses and resumes running only after it confirms that the specified CloudWatch alarm exists. If the command succeeds, no output is returned. ::
```

Comment:
Same here. The line for "This command produced no output." should be moved to after the command example where the output section is.
https://aws.github.io/aws-cli/docs_styleguide.html#command-produces-no-output